api/tests/Api/Users/ReadUserTest.php

```diff
@@ -36,13 +44,34 @@ public function testGetSingleUserIsAllowedForSelf() {
         $this->assertResponseStatusCodeSame(200);
         $this->assertJsonContains([
             'id' => $user->getId(),
-            'email' => $user->email,
-            'username' => $user->username,
-            'firstname' => $user->firstname,
-            'surname' => $user->surname,
-            'nickname' => $user->nickname,
-            'language' => $user->language,
-            'displayName' => 'Bi-Pi',
+            'displayName' => $user->getDisplayName(),
+            '_links' => [
+                'self' => [
+                    'href' => $this->getIriFor('user1manager'),
+                ],
+                'profile' => [
+                    'href' => '/profiles/'.$user->getId(),
+                ],
+            ],
+        ]);
+    }
```

Comment:
Wie wärs mit einem Test dass gewisse Properties des Users jetzt in diesem Endpoint nicht mehr verfügbar sind? Dafür könntest du vielleicht auch einfach diesen Test auf `assertJsonEquals` umstellen.

Reply:
Da hast du recht, ist gemacht bei einem test.